StaticAnalyzer/views/android/icon_analysis.py

```diff
@@ -73,3 +97,43 @@ def get_icon(apk_path, res_dir, tools_dir):
         }
     except:
         PrintException("[ERROR] Get icon function")
+
+
+def find_icon_path_zip(res_dir, icon_paths_from_manifest):
```

Comment:
Does this support both Android Studio and old eclipse projects? 
Have you checked?